test/unit/spec/nbextensions/bulkImportCell/fileTypePanel-spec.js

```diff
@@ -1,153 +0,0 @@
-define([
```

Comment:
This is a duplicate of the tests in `test/unit/spec/nbextensions/bulkImportCell/tabs/fileTypePanel-spec.js` (it was moved, but this file wasn't removed. Now it is!)